### PR TITLE
Add message about contributing to scaladoc footer

### PR DESCRIPTION
**Pull request by adamvoss**
A shameless plug encouraging involvement in the footer of the documentation:

image

### CODE REVIEW

File: src/scaladoc/scala/tools/nsc/doc/html/page/Entity.scala [hunk 1]
```diff
@@ -396,7 +396,7 @@ trait EntityPage extends HtmlPage {
 
       {
         if (Set("epfl", "EPFL").contains(tpl.universe.settings.docfooter.value))
-          <div id="footer">Scala programming documentation. Copyright (c) 2003-2016 <a href="http://www.epfl.ch" target="_top">EPFL</a>, with contributions from <a href="http://www.lightbend.com" target="_top">Lightbend</a>.</div>
+          <div id="footer">Scala programming documentation. Copyright (c) 2003-2016 <a href="http://www.epfl.ch" target="_top">EPFL</a>, with contributions from <a href="http://www.lightbend.com" target="_top">Lightbend</a>. Get involved on <a href="https://github.com/scala/scala" target="_top">GitHub</a>.</div>
```
Comment (adriaanm) [inline src/scaladoc/scala/tools/nsc/doc/html/page/Entity.scala:399]: Could we deep-link to the github page that corresponds to the documented file, using `templateToPath(tpl)`?

**Comment by SethTisue** [comment]
I like it.

on the specific wording, it sounds a bit like a general exhortation to get involved with _something_, or with Scala generally, maybe make it more targeted, something like:

```diff
-Get involved on GitHub
+Fix or improve this page on GitHub`
```

?

File: src/scaladoc/scala/tools/nsc/doc/html/page/Entity.scala (hunk 1 above)
Comment (gourlaysama) [reply src/scaladoc/scala/tools/nsc/doc/html/page/Entity.scala:399]: +1 for a direct link if possible, although I think it'is `Entity.sourceUrl` for the link-to-source feature, and `templateToPath` is for the path to the scaladoc page.

**Comment by SethTisue** [comment]
@vossad01 interested in getting this across the finish line...?

**Comment by adriaanm** [comment]
Please consider resubmitting when you get a chance to incorporate the feedback.

**Pull request closed without merging**